### ISSUE TITLE
Release workflow is broken

### ISSUE THREAD

**Issue by avdv** [tweag/rules_nixpkgs#495]
**Describe the bug**
Currently our release workflow is broken on CI, because of https://github.com/cgrindel/gha_join_jobs/issues/52

This causes the release tarball artifact to be missing.

The artifact for release 0.11.1 has been created manually, apparently without using the `release-prep` script since user information leaked into the current tarball:
```
   1   │drwxr-xr-x  0 rednaz users       0 Mar 29 19:47 rules_nixpkgs-0.11.1/
   2   │ -rw-r--r--  0 rednaz users     140 Mar 29 19:47 rules_nixpkgs-0.11.1/BUILD.bazel
   3   │ drwxr-xr-x  0 rednaz users       0 Mar 29 19:47 rules_nixpkgs-0.11.1/nix/
   4   │ -rw-r--r--  0 rednaz users    1002 Mar 29 19:47 rules_nixpkgs-0.11.1/nix/flake.nix
   5   │ -rw-r--r--  0 rednaz users    5050 Mar 29 19:47 rules_nixpkgs-0.11.1/nix/flake.lock
   6   │ -rw-r--r--  0 rednaz users   26791 Mar 29 19:47 rules_nixpkgs-0.11.1/guide.md
   7   │ -rw-r--r--  0 rednaz users     519 Mar 29 19:47 rules_nixpkgs-0.11.1/CONTRIBUTORS
   8   │ -rw-r--r--  0 rednaz users   15827 Mar 29 19:47 rules_nixpkgs-0.11.1/CHANGELOG.md
...
```
This means re-producing the tarball is not easily possible. Is this something we want to fix, or simply keep it as is since it already made its way to the BCR?

**To Reproduce**
When creating a release tag, the workflow fails; see https://github.com/tweag/rules_nixpkgs/actions/workflows/release.yaml

**Expected behavior**
The release workflow should create a draft release, including the tarball artifact when pushing a release tag.

**Environment**
Github Actions

@prednaz

**Comment by prednaz** [comment]
ah. sorry about that.

**Comment by avdv** [comment]
This was resolved as part of #565 which changed the release workflow.
